README.rst

```diff
@@ -20,118 +21,42 @@ File submission for CGAP
 Description
 ===========
 
-This is a tool for uploading files to CGAP.
-
-
-System Requirements
-===================
-
-* Python 3.6
-* Pip (Any should work)
-* Virtualenv (>=15.0.0)
-
-Setting Up a Virtual Environment (OPTIONAL)
-===========================================
-
-This is optional.
-If you do not create a virtual environment, Poetry will make one for you.
-But there are still good reasons you might want to make your own, so here
-are three ways to do it:
-
-* If you have virtualenvwrapper that knows to use Python 3.6::
-
-   mkvirtualenv myenv
-
-* If you have virtualenv but not virtualenvwrapper,
-  and you have python3.6 in your ``PATH``::
-
-   virtualenv myenv -p python3.6
-
-* If you are using ``pyenv`` to control what environment you use::
-
-   pyenv exec python -m venv myenv
-
-
-Activating a Virtual Environment
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-You should execute all actions related to this repository
-from within a virtual environment.
-
-To activate a virtual environment::
-
-   source myenv/bin/activate
-
-
-Deactivating a Virtual Environment
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-It's rarely necessary to deactivate a virtual environment.
-One is automatically deactivated if you enter another,
-and normally they have very little effect on other actions you might
-take. So it's normally safe to just leave them activated.
-
-However, if you want to deactivate an active environment, just do::
-
-   deactivate
-
-Installing Poetry in a Virtual Environment
-==========================================
-
-Once you have created a virtual environment, or have decided to just let Poetry handle that,
-install with poetry::
+This is a tool for uploading certain kinds of files to CGAP.
 
-   poetry install
+Initial support is for "metadata bundles", which are Excel files (.xls or .xlsx)
+that are accompanied by other files (such as .fastq files).
```

Comment:
instead of .fastq you could perhaps specify .fastq.gz to be consistent. 

additional note - .csv support will be added after cgap-portal#207 is merged, but not yet.